presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -2247,6 +2256,29 @@ private void assertRequestedColumnsInLayout(Plan plan, String tableName, Set<Str
         assertEquals(requestedColumnsSet, expectedRequestedColumns);
     }
 
+    private ExtendedHiveMetastore replicateHiveMetastore(DistributedQueryRunner queryRunner)
+    {
+        URI baseDir = queryRunner.getCoordinator().getBaseDataDir().resolve("hive_data").toUri();
+        HiveClientConfig hiveClientConfig = new HiveClientConfig();
+        MetastoreClientConfig metastoreClientConfig = new MetastoreClientConfig();
+        HdfsConfiguration hdfsConfiguration = new HiveHdfsConfiguration(new HdfsConfigurationInitializer(hiveClientConfig, metastoreClientConfig), ImmutableSet.of());
+        HdfsEnvironment hdfsEnvironment = new HdfsEnvironment(hdfsConfiguration, metastoreClientConfig, new NoHdfsAuthentication());
+        return new FileHiveMetastore(hdfsEnvironment, baseDir.toString(), "test");
+    }
+
+    private void appendTableParameter(ExtendedHiveMetastore metastore, String tableName, String parameterKey, String parameterValue)
+    {
+        MetastoreContext metastoreContext = new MetastoreContext(getSession().getUser(), getSession().getQueryId().getId(), Optional.empty(), Optional.empty());
+        Optional<Table> table = metastore.getTable(metastoreContext, getSession().getSchema().get(), tableName);
+        if (table.isPresent()) {
+            Table originalTable = table.get();
+            Table alteredTable = Table.builder(originalTable).setParameter(parameterKey, parameterValue).build();
+            // TODO: add a method `alterTable` in `ExtendedHiveMetastore`
```

Comment:
@gggrace14 built alterTable once. But it is highly recommended not to do it. We should lean on metastore to do the work for us. Remove this comment.

Reply:
Thanks for pointing this out. Me and Rohit were trying to figure out why this method is not implemented.